spec/system/request_for_comments_filter_system_spec.rb

```diff
@@ -37,4 +37,18 @@
     visit(request_for_comments_path)
     expect(page).to have_css('ul.pagination')
   end
+
+  it 'allows reporeting of RfCs', :js do
+    request_for_comment = create(:rfc)
+    visit(request_for_comment_path(request_for_comment))
+
+    expect do
+      accept_confirm do
+        click_on I18n.t('reports.report')
+      end
+
+      expect(page).to have_text(I18n.t('reports.reported'))
+    end.to have_enqueued_mail(ReportMailer, :report_content)
+      .with(params: {reported_content: request_for_comment}, args: [])
```

Comment:
I know, we haven't switched from controller tests to request tests, yet, but IMO, this would be good moment to start. When having a request spec file for `report#create` or for `comment#report` and `RfC#report`, the enqueued mail could be expected there, which would make this system spec more clean:

```suggestion
    click_on I18n.t('reports.report')
    expect(page).to have_text(I18n.t('reports.reported'))
```

Reply:
I created a request test and moved the email test there.

Reply:
Thanks!